content/Watsonx/CodeAssistant/101.md

```diff
@@ -0,0 +1,78 @@
+---
+title: '101: Ansible Lightspeed Environment Setup'
+timeToComplete: 15
+updated: 2023-09-13
+---
+
+# 101: Ansible Lightspeed environment setup in VS Code
+
+## Goal
+
+The goal of this lab is to install the Ansible Lightspeed extension and enable and authenticate the extension in VS Code. 
+
+## Install Ansible extension in VS Code
+
+1. Open the VS Code application.
+2. Click the **Extensions** icon on the left sidebar.
+
+    ![](./images/settings-icon.png)
```

Comment:
I was a little confused by the picture for this one.  If the instruction is to click the extensions icon on the left sidebar, should we just show a box around the extensions icon?